js/src/pages/price-benchmark/index.scss

```
@@ -1,7 +1,24 @@
 @import "@wordpress/dataviews/build-style/style.css";
 
-.gla-price-benchmark__card .app-tab-nav:first-child {
-	padding: 0 $grid-unit-40;
+.gla-price-benchmark__card {
+
+	& .app-tab-nav:first-child {
+		padding: 0 $grid-unit-40;
+	}
+
+	& .woocommerce-table.is-empty {
+		background: #fff;
+
+		.gla-price-benchmark__empty-metrics {
+			color: #757575;
+			max-width: 454px;
+			text-align: center;
+
+			p {
+				font-size: $grid-unit-20;
```

@ankitrox You're using a spacing unit for font size. I don't see any variables for `16px`. I suggest you hardcode it:
```suggestion
				font-size: 16px;
```

Please add the line height as well, which should be `20px`. `$gla-line-height-medium`

---

Thanks, added 16px value.